Jenkinsfile

```diff
@@ -381,23 +384,37 @@ pipeline {
                         unstash 'uaa-war'
                 }
                 dir('uaa-cf-release') {
-                    git changelog: false, credentialsId: 'github.build.ge.com', poll: false, url: 'https://github.build.ge.com/predix/uaa-cf-release.git', branch: 'master'
+                    git changelog: false, credentialsId: 'github.build.ge.com', poll: false, url: 'https://github.build.ge.com/predix/uaa-cf-release.git', branch: 'clz_jenkins'
                 }
```

Comment:
This should point to the master branch of uaa-cf-release. Otherwise, if we want to execute with different branches of this repo we should make it a variable.

Reply:
Good Catch....this was only during testing ....once all the changes are merged it needs to be master.
fixed it.